### DISCUSSION TITLE
npx create-next-app <app_name> --template redux doesn't create redux template

### DISCUSSION THREAD

**Discussion by hamza786504** [vercel/next.js#41372] [Help]
I want to create a redux template while creating the next js app but when I try to setup a redux template with next js like this

**npx create-next-app <app_name> --template redux**
does not create a redux template. Just create next app like this

![directories preview](https://i.stack.imgur.com/zWtUK.png)

![package.json file](https://i.stack.imgur.com/BKPDx.png)

**Comment by icyJoseph** [comment]
Hi,

There's no --template flag anymore, https://www.npmjs.com/package/create-next-app#user-content-options use:

```
npx create-next-app --example with-redux with-redux-app
```

**Comment by hamza786504** [comment]
Thanks for giving me suggestion but still doesn't work as expected
When I hit **npx create-next-app --example with-redux with-redux-app**
command. There comes two issue


1)-

Abortion installation.

npm install has failed.

node_modules folder, package lock.json and package.js file were not created.

2)-

No doubt it sets up the redux which is in typescript files infect the
whole project follows typescript instead of plain JavaScript. I want
this in plain JavaScript.

Thanks

**Comment by icyJoseph** [comment]
What `npm/node` version do you have? I think there has been some buggy ones out there. Try with latest `npm` at least.

As to TypeScript vs JavaScript, well, you could just learn something new :) but if you want JS:

```
npx create-next-app --example https://github.com/vercel/next.js/tree/077097b7f8d3d826118518a90f27cebd2b1373e6/examples/with-redux with-redux-app
```

Which is the last version of that example to ever use JS
